message.js

```diff
@@ -0,0 +1,80 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const process = require('process');
+
+const buildDeltaMessage = (oldCoverage, newCoverage) => {
+    return [
+        '',
+        '| Measure | Main branch | ' + process.env.GITHUB_REF + ' |',
+        '| --- | --- | --- |',
+        '| Coverage | ' + oldCoverage.coverage + '% | ' + newCoverage.coverage + '% |',
+        '| Total lines | ' + oldCoverage.total + ' | ' + newCoverage.total + ' |',
+        '| Covered lines | ' + oldCoverage.covered + ' | ' + newCoverage.covered + ' |',
+        '',
+        '∆ ' + (newCoverage.coverage - oldCoverage.coverage).toFixed(3),
+        ''
+    ].join('\n');
+}
+
+const buildFailureMessage = (oldCoverage, newCoverage) => {
+    return ':x: Your code coverage has been degraded :sob:' + buildDeltaMessage(oldCoverage, newCoverage);
+};
+
+const buildSuccessMessage = (oldCoverage, newCoverage) => {
+    return ':white_check_mark: Your code coverage has not been degraded :tada:' + buildDeltaMessage(oldCoverage, newCoverage);
+};
+
+const buildResultMessage = (oldCoverage, newCoverage) => {
+    if (newCoverage.coverage < oldCoverage.coverage) {
+        core.setFailed('Code coverage has been degraded');
+
+        return buildFailureMessage(oldCoverage, newCoverage);
+    }
+
+    return buildSuccessMessage(oldCoverage, newCoverage);
+}
+
+const postMessageOnPullRequest = async (message, header) => {
+    const context = github.context;
+
+    console.log(message);
```

Comment:
are we keeping this or was it for debugging?

Reply:
I find it handy to have info in the action logs :)